mesonbuild/mesonlib/universal.py

```diff
@@ -229,7 +229,6 @@ def verbose_git(cmd: T.List[str], workingdir: T.Union[str, bytes, os.PathLike],
     return p.returncode == 0
 
 def set_meson_command(mainfile: str) -> None:
-    global python_command  # pylint: disable=global-statement
```

Comment:
Is it possible to reverse these two commits? It seems odd to add a suppression comment, then immediately remove it along with the original line.